auth/server/base/src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -2428,6 +2429,7 @@ void succeed() {
             SecurityRealm.safeHandleRealmEvent(getRealmInfo().getSecurityRealm(), new RealmSuccessfulAuthenticationEvent(realmIdentity, authorizedIdentity.getAuthorizationIdentity(), null, null));
             SecurityDomain.safeHandleSecurityEvent(authorizedIdentity.getSecurityDomain(), new SecurityAuthenticationSuccessfulEvent(authorizedIdentity));
             realmIdentity.dispose();
+            log.tracef("Authentication Completed");
```

Comment:
This is just a very minor comment but since a similar message (`"Handling AuthenticationCompleteCallback: succeed"`) is already logged when handling the `AuthenticationCompleteCallback`, we can probably get away without this log message here.

Reply:
Fixed